core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -630,11 +639,20 @@ class TimeSeriesShard(val dataset: Dataset,
     indexRb.addLong(endTime)
     // Need to add 4 to include the length bytes
     indexRb.addBlob(p.partKeyBase, p.partKeyOffset, BinaryRegionLarge.numBytes(p.partKeyBase, p.partKeyOffset) + 4)
-    logger.debug(s"Added into timebucket partId ${p.partID} in shard=$shardNum partKey[${p.stringPartition}] with " +
-      s"startTime=$startTime endTime=$endTime")
+    logger.debug(s"Added into timebucket partId ${p.partID} in dataset=${dataset.ref} shard=$shardNum " +
```

Comment:
I think next time, the changing of the logs should be a separate PR.

Reply:
This was absolutely necessary for keeping sanity and debug my dev environment. With two datasets in same cluster, it is hard to distinguish which logs are for which dataset.